kDrive/AppDelegate+Scene.swift

```diff
@@ -16,22 +16,18 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import Foundation
 import kDriveCore
 import UIKit
 
 extension AppDelegate {
     func application(_ application: UIApplication,
-                     handleEventsForBackgroundURLSession identifier: String,
-                     completionHandler: @escaping () -> Void) {
-        Log.bgSessionManager("background session relaunched identifier:\(identifier)")
-        if identifier == DownloadQueue.backgroundIdentifier {
-            backgroundDownloadSessionManager.backgroundCompletionHandler = completionHandler
-        } else if identifier.hasSuffix(UploadQueue.backgroundBaseIdentifier) {
-            backgroundUploadSessionManager.handleEventsForBackgroundURLSession(identifier: identifier,
-                                                                               completionHandler: completionHandler)
-        } else {
-            completionHandler()
-        }
```

Comment:
handleEventsForBackgroundURLSession is nowhere to be found in the AppDelegate but it is still used by `backgroundDownloadSessionManager`

Reply:
It was removed here https://github.com/Infomaniak/ios-kDrive/pull/1187#discussion_r1615529793

I must admit I've not checked if background sessions are working for downloads

Reply:
re-added the old `AppDelegate+BGNSURLSession` file